### PR TITLE
fixed create user profile api, added unit test

### PR DESCRIPTION
**Pull request by lucamatsumoto**
I fixed the user profile create api. Also, I added a unit test. If this is ok, I can fix the updates as well.

### CODE REVIEW

**Comment by lucamatsumoto** [comment]
My unit test works fine on vagrant but not on Travis. Is this a database issue on Travis?

File: Vagrantfile [hunk 1]
```diff
@@ -74,7 +74,7 @@ Vagrant.configure(2) do |config|
      sudo apt-get update
      sudo apt-get install -y git nodejs nodejs-legacy npm python-pip python-dev postgresql-client postgresql postgresql-contrib postgresql-server-dev-all
      sudo npm install -g bower
-     sudo -u postgres psql postgres -c "CREATE USER vagrant;"
```
Comment (dmorina) [inline Vagrantfile]: unnecessary change?

File: crowdsourcing/serializers/utils.py [hunk 2]
```diff
@@ -9,21 +9,68 @@ class RegionSerializer(serializers.ModelSerializer):
...
+        return region
+
 
 class CountrySerializer(serializers.ModelSerializer):
+    region = RegionSerializer()
+
     class Meta:
         model = models.Country
 
+    def create(self, **kwargs):
```
Comment (dmorina) [inline crowdsourcing/serializers/utils.py]: these are pre loaded by the system, the user will select from a dropdown or sth

File: Vagrantfile (hunk 1 above)
Comment (lucamatsumoto) [inline Vagrantfile]: When I run unit test on Vagrant, it creates test_crowdsource_dev database by using vagrant account. Without adding CREATEDB, it didn't work.
